src/app/menubar/components/menubar-container/menubar-container.component.css

```diff
@@ -30,4 +30,69 @@ button {
 
 button:hover {
   background-color: #eaeaea;
+} */
+
+.bar {
+  height: 60px;
+   box-shadow: 0 2px 8px 0 rgba(0,0,0,0.09);
+   padding: 20px 40px;
+   display: flex;
+   flex-direction: row;
+   align-items: center;
+   justify-content: space-between;
+}
+
+.bar > * {
+  font-size: 15px;
+}
+
+.logo {
+  background-image: url(../../../../assets/images/GO-logo-magenta.png);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-position: 0 1.38px;
+  height: 65px;
+  width: 200px;
+  content: ' ';
+  display: block;
+  overflow: hidden;
+  margin: 0;
+}
+
+.lists {
+  display: flex;
+  align-items: center;
+}
+
+.item {
+  margin-left: 40px;
+  display: inline-block;
+}
+
+.item > a {
+  text-decoration: none;
+  color: black;
+  padding-bottom: 10px;
```

Comment:
ik zou hier ook nog een padding top in steken van 10px

Reply:
die bottom padding was eigelijk juist voor het lijntje wanneer je hovert. Waarom zou jij nog een padding-top doen?

Reply:
omdat je dan minder precies  moet klikken, bij buttons heb je meestal wel wat marge boven en onder het woord